dsgrid/cli/dsgrid_admin.py

```diff
@@ -169,8 +172,66 @@ def remove_dataset(registry_manager, dataset_id):
     registry_manager.dataset_manager.remove(dataset_id)
 
 
+def _path_cb(_, __, val):
+    return Path(val)
+
+
+@click.command()
+@click.argument("src_registry_path", type=click.Path(exists=True), callback=_path_cb)
+@click.argument("dst_registry_path", type=click.Path(exists=False), callback=_path_cb)
+@click.argument("config_file", type=click.Path(exists=True), callback=_path_cb)
+@click.option(
+    "--data-symlinks/--no-data-symlinks",
+    default=False,
+    is_flag=True,
+    show_default=True,
+    help="Make symlinks to data files (no big copy operations).",
+)
+@click.option(
+    "-r",
+    "--use-rsync",
+    default=False,
+    is_flag=True,
+    show_default=True,
+    help="Use rsync instead of copy. Mutually exclusive with --data-symlinks. Not available on "
+    "Windows.",
+)
+@click.option(
+    "-f",
+    "--force",
+    default=False,
+    is_flag=True,
+    show_default=True,
+    help="Overwrite dst_registry_path if it already exists. Only applies if use_rsync is False.",
```

Comment:
NIT, if I am interpreting this correctly, you can either call:
`--data-symlinks` or `--use-rsync`+ optionally`--no-data-symlinks`? And `-f` only works with the former case (i.e. with `--data-symlinks`)? If so, what's the use of having both `--data-symlinks` and `--no-data-symlinks`?

Reply:
There should be 3 options:
1. (default) data_symlinks=false use_rsync=false. Copy all data to a new directory. Takes a long time.
2. data_symlinks=true use_rsync=false. Copy all configs to a new directory. Create symlinks to the data files. Very fast.
3. data_symlinks=false use_rsync=true. Run `rsync src dst`. This will skip files that already exist. Slow the first time, then fast. But also kinda pointless if you are filtering data on the copy.